src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/monitor-services/monitor-services-running-kubernetes.mdx

```diff
@@ -0,0 +1,342 @@
+          label.app: sampleapp
+
+      integrations:
+        - name: nri-redis
+          env:
+            HOSTNAME: ${discovery.ip}
+            PORT: 6379
+```
+
+<Callout variant="tip">
+  Notice we are specifying `--tls --port 10250`. Earlier versions of the integration might have worked without this, but it is mandatory since v3 as we now connect to the Kubelet using the nodeIP rather than `localhost`.
+</Callout>
+
+Integrations targeting other pods should have their own section next to `redis-sampleapp.yaml`.
+
+Integrations are standalone binaries, and are executed by the infrastructure agent that is included in the `newrelic-nrk8s-kubelet-xxxxx` pods. Config files are deployed to all pods of the `nrk8s-kubelet` DaemonSet, but discovery ensures that each pod will only target service pods that are scheduled in the same node as that particular `nrk8s-kubelet` pod. If an instance of the `nrk8s-kubelet` DaemonSet does not have any pods matching the specified labels, the integration will not be executed by that instance.
+
+#### (Legacy v2) Add the configuration to `values.yaml` [#configure-values-v2]
+
+<CollapserGroup>
+  <Collapser
+    id="example-config-map"
+    title="Example configuration for v2"
+  >
+    <Callout variant="tip">
+      This format applies to the legacy Kubernetes integration v2. For newer versions, check the section [above](#configure-values).
+    </Callout>
+
+    This example shows the YAML config for the Redis integration added to the `values.yml` file of the `nri-bundle` chart.
+
+    ```yaml
+    newrelic-infrastructure:
+    integrations_config:
+    - name: nri-redis.yaml
+    data:
+    discovery:
+    command:
+    # Run NRI Discovery for Kubernetes
+    # https://github.com/newrelic/nri-discovery-kubernetes
+    exec: /var/db/newrelic-infra/nri-discovery-kubernetes --tls --port 10250
+    match:
+    label.app: redis
+    integrations:
+    - name: nri-redis
+    env:
+    # using the discovered IP as the hostname address
+    HOSTNAME: ${discovery.ip}
+    PORT: 6379
+    labels:
+    env: test
+    ```
+  </Collapser>
+</CollapserGroup>
+
+#### Verify the integration is working
+
+Go to **[one.newrelic.com](http://one.newrelic.com) > Infrastructure**, select **Third party services**, and then select the service's dashboard. You should see data being reported.
+
+If you do not see the data there, the integration might be missing some parameter that it requires to run, or might not be able to reach the target service. You can get the logs of the integration by running:
+
+```shell
+kubectl logs -n <var>newrelic</var> newrelic-nrk8s-kubelet-<var>xxxxx</var> agent
+```
+
+Make sure to select the particular pod of the `nrk8s-kubelet` DaemonSet that is scheduled next to the pod that the integration should be targeting. You can check which instance is running on which node by running the following command:
+
+```shell
+kubectl get pods -n newrelic -o wide -l app.kubernetes.io/component=kubelet
```

Comment:
In the logs also autodiscovery info can be found, but maybe it is too advance to be put here

Reply:
I put a command to get the agent logs just above this paragraph, is that what you are referring to?

Reply:
resolved